### ISSUE TITLE
Option not to remove buttons from modern GTK applications

### ISSUE THREAD

**Issue by HubKing** [hardpixel/unite-shell#317]
I think hiding the three buttons from modern GTK applications that have, I don't know what its technical term is, the thick top tool bar (e.g., Files, gedit, etc) when their window is maximised is useless, because their window title bar is also a toolbar so it cannot be fully hidden anyway. Hiding the three buttons alone may save some horizontal space, but since today's monitors are wide and the top tool bar is often not crowded, so to me, it seems meaningless. More over, it causes problems with full-screen apps like [Celluloid](https://github.com/celluloid-player/celluloid). Celluloid shows the top tool bar when it is in full-screen mode, but if I enable Unite, it removes the three buttons from full-screen Celluloid.

Maybe this is possible? But I could not find the correct setting to achieve this. It seems that it needs re-login to apply changed window/button options, so it took a lot of time to experiment and I still could not find it.

**Comment by natewind** [comment]
It makes the UX more unified though. The buttons are always in the same place.

**Comment by bluesceada** [comment]
I am with HubKing. In my opinion it is not unified anyway, like he says. But when maximizing these applications and suddenly the minimize/maximize/close buttons are not within the titlebar (which cannot be removed), it is confusing.

**Comment by dedeweb** [comment]
I agree with HubKing and bluesceada, this would be a great improvement

**Comment by bkurdali** [comment]
I would love this too! I use the extension to have a fairly 'stock' gnome experience, I just love to have certain applications (blender) without a title bar when maximized, and don't need the buttons at all. I don't really need to change applications with CSDs since it saves not much space anyway.
